src/index.html

```diff
@@ -11,7 +11,213 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
+      <fieldset>
+        <legend>Personal information</legend>
+        <label 
+          for="surename" 
+          class="fieldsetItem"
+        >
+          Surname:
+          <input 
+            type="text" 
+            name="surename" 
+            autocomplete="off"
+            id="surename"
+          >
+        </label>
+        <label 
+          for="name" 
+          class="fieldsetItem"
+        >
+          Name:
+          <input 
+            type="text" 
+            name="name" 
+            autocomplete="off" 
+            required="" 
+            id="name"
+          >
+        </label>
+        <label 
```

Comment:
add en empty line between _multiline_ siblings. Apply all over the page
```suggestion
        </label>
        
        <label 
```